### PR TITLE
Change dotenv-linter installation way

### PR DESCRIPTION
**Pull request by DDtKey**
Now we can use the uniform api to install the `reviewdog` and `dotenv-linter`! :relaxed:

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,7 @@ ENV REVIEWDOG_VERSION v0.10.1
 
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 RUN apk add --no-cache git
-RUN wget https://github.com/dotenv-linter/dotenv-linter/releases/download/$DOTENV_LINTER_VERSION/dotenv-linter-alpine-x86_64.tar.gz -O - -q | tar -xzf -
-RUN mv dotenv-linter /usr/local/bin/
+RUN wget -q -O - https://raw.githubusercontent.com/dotenv-linter/dotenv-linter/master/install.sh | sh -s -- -b /usr/local/bin/ $DOTENV_LINTER_VERSION
 RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /usr/local/bin/ $REVIEWDOG_VERSION
```
Comment (mgrachev) [inline Dockerfile:9]:
```suggestion
RUN wget -q -O - https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /usr/local/bin/ $REVIEWDOG_VERSION
```

**Comment by mgrachev** [comment]
@DDtKey 👍

I think we should bump the vesrion of `dotenv-linter` to v2.1.0.

**Comment by DDtKey** [comment]
> I think we should bump the vesrion of `dotenv-linter` to v2.1.0.

I also wanted to, but I thought that the version will be updated automatically, on a schedule using github-actions.
Until we publish release 2.1.0 CI will not pass successfully.
But I agree!

**Comment by mgrachev** [comment]
@DDtKey Thank you! 🙏

**Pull request merged**